reana_server/reana_admin.py

```diff
@@ -214,39 +209,102 @@ def import_users(ctx, admin_access_token, file_):
 @admin_access_token_option
 @click.option(
     '--id',
+    'id_',
     help='The id of the user.')
 @click.option(
     '-e',
     '--email',
     help='The email of the user.')
-def token_grant(id, email, admin_access_token):
-    """."""
-    user_id = 'dc4aef21-c324-4fd5-8480-c724de1f0c06'
-    user_email = 'john.doe@google.com'
-    user_granted_token = 'c0fa47fa00ae4013a13fd7n'
-    # create a token in UserToken table and activate it
-    # create an audit log entry
-    # send notification to user by email
-    click.secho(f'Token for user {user_id} ({user_email}) granted.\n'
-                f'\nToken: {user_granted_token}', fg='green')
+@click.option(
+    '-f',
+    '--force',
+    is_flag=True,
+    default=False,
+    help='Grant token even if user has a revoked access token.')
+def token_grant(admin_access_token, id_, email, force):
+    """Grant a token to the selected user."""
+    try:
+        admin = User.query.filter_by(id_=ADMIN_USER_ID).one_or_none()
+        if admin_access_token != admin.access_token:
+            raise ValueError('Admin access token invalid.')
+        user = _get_user_by_criteria(id_, email)
+        error_msg = None
+        if not user:
+            error_msg = f'User {id_ or email} does not exist.'
+        elif user.access_token:
+            error_msg = (f'User {user.id_} ({user.email}) has already an'
+                         ' active access token.')
+        elif (not force and user.access_token_status in
+              [UserTokenStatus.revoked.name, None]):
+            error_msg = (f'User {user.id_} ({user.email}) access token status'
+                         f' is {user.access_token_status}, if you want to'
+                         ' proceed in any case use --force option.')
```

Comment:
This means we will always have to use `--force` right? If that is the case, I think we should not use force but rather a confirmation such as https://click.palletsprojects.com/en/7.x/prompts/#confirmation-prompts, what do you think?

Reply:
Not always, only when the user has not requested the token or the token has been revoked, because they are not usual actions.
For the typical workflow, when the user has requested the token, you don't need to use `--force`.
Said that, the confirmation prompt seems like a good idea for the "force" cases, I didn't know this feature.

Reply:
Fixed, now it displays something like:
```console
$ flask reana-admin token-grant -e johndoe@cern.ch
User xxx-aaa-bbb (johndoe@cern.ch) access token status is None, do you want to proceed? [y/N]: N
Grant token aborted.
```